src/main/java/core/basesyntax/impl/ConvertToCsvServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ConvertToCsvService;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ConvertToCsvServiceImpl implements ConvertToCsvService {
+    private static final String[] HEADER = {"fruits", "quantity" + System.lineSeparator(),};
+    private static final int STORAGE_SIZE = Storage.fruits.size();
+
+    @Override
+    public List<String[]> convertedList() {
+        List<String[]> result = new ArrayList<>();
+        result.add(HEADER);
+        Object[] keys = Storage.fruits.keySet().toArray();
```

Comment:
you convert a string to an object, and after this - convert this object to a string. This is a bad solution. 
You can go through every single pair in the map using Map.Entry.
Example:
`   for (Map.Entry<String, Integer> entry : map.entrySet()) {
        }`